crates/swc_ecma_transforms_module/src/system_js.rs

```diff
@@ -574,9 +589,12 @@ impl Fold for SystemJs {
                 _ => Expr::MetaProp(meta_prop_expr),
             },
             Expr::Await(await_expr) => {
+                let await_expr = await_expr.fold_children_with(self);
```

Comment:
This seems like a double visit

Reply:
Yeah, you're right. But only here, not in `fold_prop`